python/cuml/cluster/kmeans.pyx

```diff
@@ -524,16 +553,18 @@ class KMeans(Base):
                 <double*> input_ptr,            # srcdata
                 <size_t> self.n_rows,           # n_samples (rows)
                 <size_t> self.n_cols,           # n_features (cols)
-                <int> 1,                        # distance metric as L2-norm/euclidean distance: @todo - support other metrics
+                <int> 1,                        # distance metric as L2-norm/euclidean distance: @todo - support other metrics # noqa
                 <double*> preds_ptr,            # transformed output
                 <int> self.verbose)
         else:
-            raise TypeError("supports only float32 and float64 input, but input of type '%s' passed." % (str(self.gdf_datatype.type)))
+            raise TypeError('KMeans supports only float32 and float64 input,'
+                            'but input type ' + str(self.gdf_datatype.type) +
+                            ' passed.')
 
         self.handle.sync()
         preds_gdf = cudf.DataFrame()
         for i in range(0, self.n_clusters):
-            preds_gdf[str(i)] = preds_data[i:self.n_rows * self.n_clusters:self.n_clusters]
+            preds_gdf[str(i)] = preds_data[i:self.n_rows * self.n_clusters:self.n_clusters] # noqa
```

Comment:
Should just fix the formatting instead of noqa here

Reply:
Going to follow a full pep8 fixing PR (and that line is annoying to fix :P)